components/public-api-server/pkg/server/server.go

```diff
@@ -57,6 +61,19 @@ func Start(logger *logrus.Entry, version string, cfg *config.Configuration) erro
 		return fmt.Errorf("failed to read cipherset from file: %w", err)
 	}
 
+	var redisClient *redis.Client
+	if cfg.RedisAddress != "" {
+		redisClient = redis.NewClient(&redis.Options{
+			Addr: cfg.RedisAddress,
+		})
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+		err := redisClient.Ping(ctx).Err()
```

Comment:
I haven't dug into this, but is the Redis client reconnecting? If so, we shouldn't need to do a ping here

Reply:
Sure, but this only checks at connection time (server startup time), not subsequently so the value we get from this remains limited